src/style.css

```
@@ -1,3 +1,86 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+:root {
+  --active-collor: #00acdc;
+  --background-collor: #fff;
+  --link-collor: #000;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background-color: var(--background-collor);
+}
+
+.logo {
+  position: absolute;
+  left: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  position: relative;
+  margin: 0 10px;
+  color: var(--link-collor);
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  color: var(--active-collor);
+}
+
+.nav__list :first-child .nav__link {
+  margin-right: 10px;
+  margin-left: 0;
+}
+
+.nav__list :last-child .nav__link {
+  margin-left: 10px;
+  margin-right: 0;
+}
```

we can do it using only 1 class
```suggestion
.nav_link:not(:first-child) {
  margin-left: 20px;
}
```